CHANGES

```diff
@@ -1,11 +1,163 @@
-
-* (in development)
+* 4.1.2 (Jan 27, 2022)
+    * Invalid OCSP certificates should raise ConnectionError on failed validation (#1907)
+    * Added retry mechanism on socket timeouts when connecting to the server (#1895)
+    * LMOVE, BLMOVE return incorrect responses (#1906)
+    * Fixing AttributeError in UnixDomainSocketConnection (#1903)
+    * Fixing TypeError in GraphCommands.explain (#1901)
+    * For tests, increasing wait time for the cluster (#1908)
+    * Increased pubsub's wait_for_messages timeout to prevent flaky tests (#1893)
+    * README code snippets formatted to highlight properly (#1888)
+    * Fix link in the main page (#1897)
+    * Documentation fixes: JSON Example, SSL Connection Examples, RTD version (#1887)
+    * Direct link to readthedocs (#1885)
+* 4.1.1 (Jan 17, 2022)
+    * Add retries to connections in Sentinel Pools (#1879)
+    * OCSP Stapling Support (#1873)
+    * Define incr/decr as aliases of incrby/decrby (#1874)
+    * FT.CREATE - support MAXTEXTFIELDS, TEMPORARY, NOHL, NOFREQS, SKIPINITIALSCAN (#1847)
+    * Timeseries docs fix (#1877)
+    * get_connection: catch OSError too (#1832)
+    * Set keys var otherwise variable not created (#1853)
+    * Clusters should optionally require full slot coverage (#1845)
+    * Triple quote docstrings in client.py PEP 257 (#1876)
+    * syncing requirements (#1870)
+    * Typo and typing in GraphCommands documentation (#1855)
+    * Allowing poetry and redis-py to install together (#1854)
+    * setup.py: Add project_urls for PyPI (#1867)
+    * Support test with redis unstable docker (#1850)
+    * Connection examples (#1835)
+    * Documentation cleanup (#1841)
+* 4.1.0 (Dec 26, 2021)
+    * OCSP stapling support (#1820)
+    * Support for SELECT (#1825)
+    * Support for specifying error types with retry (#1817)
+    * Support for RESET command since Redis 6.2.0 (#1824)
+    * Support CLIENT TRACKING (#1612)
+    * Support WRITE in CLIENT PAUSE (#1549)
+    * JSON set_file and set_path support (#1818)
+    * Allow ssl_ca_path with rediss:// urls (#1814)
+    * Support for password-encrypted SSL private keys (#1782)
+    * Support SYNC and PSYNC (#1741)
+    * Retry on error exception and timeout fixes (#1821)
+    * Fixing read race condition during pubsub (#1737)
+    * Fixing exception in listen (#1823)
+    * Fixed MovedError, and stopped iterating through startup nodes when slots are fully covered (#1819)
+    * Socket not closing after server disconnect (#1797)
+    * Single sourcing the package version (#1791)
+    * Ensure redis_connect_func is set on uds connection (#1794)
+    * SRTALGO - Skip for redis versions greater than 7.0.0 (#1831)
+    * Documentation updates (#1822)
+    * Add CI action to install package from repository commit hash (#1781) (#1790)
+    * Fix link in lmove docstring (#1793)
+    * Disabling JSON.DEBUG tests (#1787)
+* 4.1.0-rc2 (Dec 09, 2021)
```

Comment:
Can we (for all releases > 4.0) merge rc (or beta) builds into their version?  This means 4.1.0 would have all of rc1, rc2, etc.